tests/test_accession_and_clustering.py

```diff
@@ -61,9 +60,17 @@ def tearDown(self) -> None:
         shutil.rmtree(self.processing_folder, ignore_errors=True)
         self.mongo_db.drop_database(self.accessioning_database_name)
 
+
+    def download_test_files(self):
+        download_file_name = os.path.basename(self.download_url)
+        snapshot_download_command = (f'bash -c "cd {self.download_target_dir} && curl -O {self.download_url} && '
```

Comment:
Not sure why we're using curl to retrieve test data. I know that's what was used before but if you're going to rewrite it I think it's unnecessary.

Reply:
refactored